source/includes/_tipser-elements.md

```diff
@@ -299,7 +299,61 @@ The `onError` event handler is used with `useDefaultErrorHandler` config option.
 
 <aside class="warning">This section requires simplification. It isn't clear how to use event handler with combination of this configuration</aside>
 
+## Customizing Tipser Elements Styles ##
+
+Tipser Elements are the "building blocks" designed to fit your page as much as possible. We created the styling in a way that delivers a nice look & feel from the start, but also allows you to change them easily to fit your unique sense of style. For example, all elements' `font-family` and `font-size` attributes are set to inherit them from the host page. If you need to change some other styles, please overwrite the certain CSS classes.
+
+### Product Card ###
+
+The Product Card is an item used for displaying single `Collection` or `Store` component, among others. The font-family used in the description section of the Product Card is inherited from your website's styles, and the font-size is expressed in the relative `em` units controlled in `.te-product-card` class. The default value used there is `12px`, which you can easily change by adding to your CSS the following style:
```

Comment:
The Product Card is an item used for displaying single `Collection` or `Store` component

maybe:

The Product Card is an item used for displaying **a single tile** in a `Collection` or `Store` component